pkg/web_css/pubspec.lock

```diff
@@ -1,5 +1,5 @@
 # Generated by pub
-# See https://dart.dev/tools/pub/glossary#lockfile
+# See https://www.dartlang.org/tools/pub/glossary#lockfile
```

Comment:
When did this change in `pub` client? The URL seems to be off, as we will redirect it first to pub.dev and then away from the site...